server/routes/user.py

```diff
@@ -74,7 +78,37 @@ def index():
     return "Data Commons </br> <a href='/user/auth/login'><button>Login</button></a>"
 
 
+@bp.route('/upload', methods=['POST'])
+@login_is_required
+def upload():
```

Comment:
nit: upload is a bit vague, maybe upload_import?

Reply:
Done